### PR TITLE
Adding initial Service Catalog acceptable use terms

### LINKED ISSUE #800: DOC: Add service catalog documentation
#### Location of the Documentation

There should be a new documentation domain called 'scientific computing' with sub parts for using resources in sc.sageit.org, which is accessed through Synapse. This is currently missing.

#### Problem

We need a space for service catalog documentation that is general and not specific to a project.

#### Suggested Fix

I'll start with some security docs.

### PR DESCRIPTION
**Pull request by ahayden**
This PR adds the initial version of the acceptable use terms for users of the service catalog. 
Depends on https://github.com/Sage-Bionetworks/synapseDocs/pull/801

- [x] Fixes #800 
- [x] If you are creating a page, did you follow the instructions [here](https://github.com/Sage-Bionetworks/synapseDocs#creating-a-page)?
- [x] Did you follow the [style guide](https://github.com/Sage-Bionetworks/synapseDocs#style-guide)?
- [x] Did you include a description of your changes or additions in this pull request?
- [ ] (Optional) Did you examine your build locally (instructions [here](https://help.github.com/en/github/working-with-github-pages/))

### CODE REVIEW

**Review by tthyer — APPROVED** [review]
Approving because I've previously reviewed the text. I thought that a decision had come out of a previous meeting to isolate any documentation to the specific, limited external groups that we are first opening the SC too, and that this document would fall under that category? But the plan may have been updated. @ryanluce72, was a change made?

**Comment by ahayden** [comment]
We had a discussion yesterday with @ryanluce72 about where to house these kinds of documents, in public Confluence pages, etc. but eventually landed on using Synapse docs. Since external use of Service Catalog is limited to a single community in the near term, we can keep this kind of thing in a community portal instead.

File: _articles/sc-acceptable-use.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Synapse Scientific Computing Acceptable Use
+layout: article
+excerpt: Computing resources through Synapse requires agreement with appropriate use
+category: scientific-computing
```
Comment (kelshmo) [inline _articles/sc-acceptable-use.md:5]: This is not a category in the docs - therefore this page will not get surfaced on the website when the PR is approved. Is that behavior desired? The possible categories are the sections on [this page](https://docs.synapse.org/articles/article_index.html) and the syntax for the yaml header is determined from the list [here](https://github.com/Sage-Bionetworks/synapseDocs/tree/gh-pages/_categories).

Comment (kelshmo) [reply _articles/sc-acceptable-use.md:5]: I see the other PR for this category.

**Comment by kelshmo** [comment]
As the Service Catalog is not usable/accessible by all Synapse users - I do not think the documentation should live here.

**Comment by ahayden** [comment]
Closing as the docs should not be housed here

**Pull request closed without merging**